### PR TITLE
refseq_Mus_musculus update

### PR DESCRIPTION
**Pull request by emmetaobrien**
This PR updates the `refseq_Mus_musculus` dataset again as the source dataset updated twice in quick succession.

### CODE REVIEW

**Review by tkkuehn — COMMENTED** [review]
I confirmed that I can get both files in the dataset. A quick question before approving though.

File: .gitmodules [hunk 1]
```diff
@@ -226,3 +223,6 @@
 [submodule "projects/braincode_EpLink"]
 	path = projects/braincode_EpLink
 	url = https://github.com/CONP-PCNO/braincode_EpLink
+[submodule "projects/refseq/refseq_Mus_musculus"]
+	path = projects/refseq/refseq_Mus_musculus
+	url = https://github.com/emmetaobrien/refseq_Mus_musculus.git
```
[the remark this answers is not in the document]
Comment (emmetaobrien) [reply .gitmodules:228]: No we don't, that was an error and I will correct it now.

[the remark this answers is not in the document]
Comment (emmetaobrien) [reply .gitmodules]: Submodule now updated to point to the version in conpdatasets.

**Review by tkkuehn — CHANGES_REQUESTED** [review]
There appears to be something wrong with the subdataset now -- `datalad get` any of the two files does nothing and `git-annex whereis` on those files prompts for a `git annex init`

**Comment by tkkuehn** [comment]
Ah, I think the issue is the `git-annex` branch hasn't been pushed to github

**Comment by emmetaobrien** [comment]
This now looks to be fixed.

**Pull request merged**